src/main/java/org/broadinstitute/hellbender/utils/GenotypeUtils.java

```diff
@@ -34,13 +32,14 @@ public static GenotypeCounts computeDiploidGenotypeCounts(final VariantContext v
                                                               final boolean roundContributionFromEachGenotype){
```

Comment:
Can you briefly explain the purpose of these (somewhat extensive) changes to computeDiploidGenotypeCounts()?

Reply:
It's a refactoring to handle the counts in terms of how many ref alleles are in the genotype rather than matching alts because it makes the het-non-ref (e.g. 1/2) case not a special case anymore.  Also `doMultiallelicMapping` is not super intuitive.